internal/testutils/feature.go

```diff
@@ -109,3 +118,15 @@ func kernelVersion(tb testing.TB) internal.Version {
 	}
 	return v
 }
+
+func ignoreKernelVersionCheck(tb testing.TB) bool {
```

Comment:
Could you add a test for this function?

Reply:
hmm writing test for this function seems tricky ( also less valuable ) because:
1. It uses `os.Getenv` . I can use [T.Setenv()](https://pkg.go.dev/testing#T.Setenv) but that will block us from running parallel tests.
2. I will have to inject mocked `testing.TB`. 

I can solve 2 by changing function signature to `ignoreKernelVersionCheck(tbName string)`. This new name looks more sensible to me. But 1 will still be troublesome.

Reply:
TIL about T.Setenv, cool! Both 1 and 2 seem like good solutions to me.

Reply:
I added the tests.